src/Symfony/Component/Security/Http/RememberMe/AbstractRememberMeServices.php

```diff
@@ -293,7 +293,15 @@ protected function cancelCookie(Request $request)
             $this->logger->debug(sprintf('Clearing remember-me cookie "%s"', $this->options['name']));
         }
 
-        $request->attributes->set(self::COOKIE_ATTR_NAME, new Cookie($this->options['name'], null, 1, $this->options['path'], $this->options['domain']));
+        $request->attributes->set(self::COOKIE_ATTR_NAME, new Cookie(
+            $this->options['name'],
+            null,
+            1,
+            $this->options['path'],
+            $this->options['domain'],
+            $this->options['secure'],
+            $this->options['httponly']
+        ));
```

Comment:
> Why? It would make a really long line (difficult to read). 207 chars, which is a "warning sign" under PSR-2, which recommends to split such lines.

We've got wide monitors and don't mind long lines ;)

but seriously, readability is a personal preference and we tend NOT to brake lines in such cases in the rest of codebase.

> BTW, is it possible to "restart" fabbot.io?

Not as far as I know.